packages/vuetify/src/components/VBanner/VBannerActions.scss

```diff
@@ -0,0 +1,29 @@
+// Imports
+@import './index';
+
+.v-banner__actions {
```

Comment:
`.v-banner-actions`?
Also same for thumbnail and content

Reply:
They are children of v-banner and under BEM should be elements.